core/src/avm2/globals/Date.as

```diff
@@ -1,5 +1,461 @@
+        }
+
+        public function get seconds():Number {
+            return this.AS3::getSeconds();
+        }
+        public function set seconds(value:Number):void {
+            this.AS3::setSeconds(value);
+        }
+
+        public function get milliseconds():Number {
+            return this.AS3::getMilliseconds();
+        }
+        public function set milliseconds(value:Number):void {
+            this.AS3::setMilliseconds(value);
+        }
+
+        public function get fullYearUTC():Number {
+            return this.AS3::getUTCFullYear();
+        }
+        public function set fullYearUTC(value:Number):void {
+            this.AS3::setUTCFullYear(value);
+        }
+
+        public function get monthUTC():Number {
+            return this.AS3::getUTCMonth();
+        }
+        public function set monthUTC(value:Number):void {
+            this.AS3::setUTCMonth(value);
+        }
+
+        public function get dateUTC():Number {
+            return this.AS3::getUTCDate();
+        }
+        public function set dateUTC(value:Number):void {
+            this.AS3::setUTCDate(value);
+        }
+
+        public function get hoursUTC():Number {
+            return this.AS3::getUTCHours();
+        }
+        public function set hoursUTC(value:Number):void {
+            this.AS3::setUTCHours(value);
+        }
+
+        public function get minutesUTC():Number {
+            return this.AS3::getUTCMinutes();
+        }
+        public function set minutesUTC(value:Number):void {
+            this.AS3::setUTCMinutes(value);
+        }
+
+        public function get secondsUTC():Number {
+            return this.AS3::getUTCSeconds();
+        }
+        public function set secondsUTC(value:Number):void {
+            this.AS3::setUTCSeconds(value);
+        }
+
+        public function get millisecondsUTC():Number {
+            return this.AS3::getUTCMilliseconds();
+        }
+        public function set millisecondsUTC(value:Number):void {
+            this.AS3::setUTCMilliseconds(value);
+        }
+
+        public function get time():Number {
+            return this.AS3::getTime();
+        }
+        public function set time(value:Number):void {
+            this.AS3::setTime(value);
+        }
+
+        public function get timezoneOffset():Number {
+            return this.AS3::getTimezoneOffset();
+        }
+
+        public function get day():Number {
+            return this.AS3::getDay();
+        }
+
+        public function get dayUTC():Number {
+            return this.AS3::getUTCDay();
+        }
+
+
+        prototype.setTime = function(t:* = undefined):Number {
+            var d:Date = this;
+            return d.AS3::setTime(t);
+        }
+        prototype.valueOf = function():* {
+            var d:Date = this;
+            return d.AS3::valueOf();
+        }
+        prototype.toString = function():String {
+            var d:Date = this;
+            return d.AS3::toString();
```

Comment:
can you make sure that this generates proper bytecode? (I think it should, given globals.as, but let's make sure just in case)

Reply:
I can't open this class in JPEXS. I did test that `Date.prototype.toString.call()` works. (But somehow it doesn't in Flash, which I don't understand)